Improve descriptions tag (#2772)
near/docs: docs/smart-contracts/anatomy/crosscontract.md

---
id: crosscontract
title: Cross-Contract Calls

> description: "Learn cross-contract calls on NEAR to interact asynchronously between contracts and handle callbacks."

Reviewer: ```suggestion
description: "Contract can interact with other contracts on the network"
```
